CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented here.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/) and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## [3.2.2] - 2021-05-28
+
+This release only contains internal changes and documentation fixes. Boring :o
+
+### Fixed
+
+- **Deprecated Sass features**: The division in Sass with the slash operator [was deprecated](https://sass-lang.com/d/slash-div).
```

Comment:
Given that [`math.div`](https://sass-lang.com/documentation/modules/math#div) was added in Sass 1.33.0, which was only released 8 days ago (20/5/2021), wouldn't this be a breaking change for people who haven't updated their Sass yet?
At least we can update the peer dependency on Sass (and use `>` instead of `^`, as it's a patch version ugrade), which will prevent updating attractions unless the Sass peer dep version is met ([npm v7](https://github.blog/2021-02-02-npm-7-is-now-generally-available/#peer-dependencies)), or at least show a warning (npm v4-6).

Reply:
Right, I misunderstood the `^` and `~` symbols